dcpy/test/connectors/edm/test_packaging.py

```diff
@@ -0,0 +1,67 @@
+import pytest
+import os
+from dcpy.connectors.edm import publishing, packaging
+
+
+def test_bucket_empty(create_buckets, mock_data_constants):
+    """Sanity check that there are no packaged versions from previous tests
+    or actual data."""
+
+    assert (
+        packaging.get_packaged_versions(
+            product=mock_data_constants["TEST_PRODUCT_NAME"]
+        )
+        == []
+    )
+
+
+def test_package_fails(create_buckets, create_temp_filesystem, mock_data_constants):
+    # packaging should fail when version is not published
+    with pytest.raises(AssertionError):
```

Comment:
Should we also assert on the error message, to make sure we're catching the error that we expect?